src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -91,6 +95,29 @@ protected static void printDecoratedExceptionMessage(final PrintStream ps, final
         return packageList;
     }
 
+
+    /**
+     * Reads from the given command-line arguments, pulls out configuration options,
+     * and initializes the configuration for this instance of Main.
+     * Returns the list of arguments without the config file option and config file path.
+     * @param args The Array of command-line arguments.
+     * @return A new Array of command-line arguments with the config file option and config file path removed.
+     */
+    protected String[] parseArgsForConfigSetupAndGetNewArgs(final String[] args) {
```

Comment:
Sounds good - I'll add it to the `Main` javadoc, and I'll update the method with some more info.

@magicDGS The class path loading variable (see the next commit) will solve class path loading.